components/src/ProductListLoadmore/productlistloadmore.stories.tsx

```diff
@@ -1,15 +1,22 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { MemoryRouter } from 'react-router';
-import productsData from './MockHttpResponses/product_list_pagination_response.json'
+import productsData from './MockHttpResponses/GET/productData_response.json';
 
 import ProductListLoadMore from './productlistloadmore';
+import { mockProductListLoadMoreFromSearchResponse } from './productlistloadmore.api.mocks';
+import { searchLookup } from '../utils/CortexLookup';
 
-function handleDataChange(){}
-function onLoadMore(){}
+function handleDataChange(updateItems){
+  alert(`handleDataChange called to update container items.  Check console to see the value that it is given.`);
```

Comment:
Is this to simulate the action of a call being made?

Reply:
Yeah that was the idea.  To show how the component is to be used.  With the README storybook plugin it would be more clear

Reply:
Okay makes sense. Thank you for the info @aChanEP! We'll add the further storybook enhancements to our backlog.